ui/ui-expenses/src/screens/list/list.reactions.js

```diff
@@ -1,9 +1,32 @@
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
 import pica from 'pica/dist/pica'
 
+export const addListener = (action, store, { window }) => {
+  window.addEventListener('paste', pasteEvent => store.dispatch({ type: '@@ui/ON_PASTE', payload: pasteEvent }))
+}
+
+export const removeListener = (action, store, { window }) => {
+  window.removeEventListener('paste')
```

Comment:
You have to remove the function: https://developer.mozilla.org/fr/docs/Web/API/EventTarget/removeEventListener